codebundles/aws-c7n-network-health/runbook.robot

```diff
@@ -0,0 +1,118 @@
+*** Settings ***
+Metadata            Author   saurabh3460
+Metadata            Supports    AWS    Tag    CloudCustodian
+Metadata            Display Name    AWS network health
+Documentation        Check AWS network health.
+Force Tags    Tag    AWS    security-group    network
+
+Library    RW.Core
+Library    RW.CLI
+Library    CloudCustodian.Core
+
+Suite Setup    Suite Initialization
+
+
+*** Tasks ***
+List security group that allow public IP access in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` 
```

Comment:
This title could be improved. 

"List Publicly Accessible Security Groups in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}`"